src/components/FormTracker/index.js

```diff
@@ -1,12 +1,29 @@
-import React from 'react';
+import React, { useState } from 'react';
 import { number, string } from 'prop-types';
 import firebase from '../../firebase';
 
 export const FormTracker = ({ age, email, id, name }) => {
+  const [edit, setEdit] = useState(true);
```

Comment:
why is this defaulting to `true`?

Reply:
I currently have disabled={edit} below so have it disabled when it shows the user and when you click edit then you can add your input. Will change this though as the UI looks horrific - currently looking at inline editable UI